### ISSUE TITLE
[Bug] design_decor:breaker_switch is not craftable

### ISSUE THREAD

**Issue by ninjaeon** [DrMango14/Create-Design-n-Decor#22]
design_decor:breaker_switch is not craftable, despite appearing in recipe viewer.

Currently on design_decor-0.4_fabric+1.20.1.jar

![image](https://github.com/DrMango14/Create-Design-n-Decor/assets/10052279/f5a0af76-1ea0-46a3-bc00-73107a4eb1ba)
![image](https://github.com/DrMango14/Create-Design-n-Decor/assets/10052279/650948e0-0cc7-43a2-b532-40398ad215b5)

Full Log:
https://mclo.gs/ZjBaNvf

**Comment by KN13KOMETA** [comment]
I assume it doesnt work due mod uses "forge" tags.
Some examples:
https://github.com/DrMango14/Create-Design-n-Decor/blob/484c3624230408e074847b1a65be522efb9fc8a7/src/main/resources/data/design_decor/recipes/crafting/breaker_switch.json#L17 https://github.com/DrMango14/Create-Design-n-Decor/blob/484c3624230408e074847b1a65be522efb9fc8a7/src/main/resources/data/design_decor/recipes/crafting/gas_tank.json#L11 https://github.com/DrMango14/Create-Design-n-Decor/blob/484c3624230408e074847b1a65be522efb9fc8a7/src/main/resources/data/design_decor/recipes/stonecutting/blank_sign.json#L5
Personally I made datapack ([design_decor_fabric_fix.zip](https://github.com/DrMango14/Create-Design-n-Decor/files/15379350/design_decor_fabric_fix.zip)) to fix it.

**Comment by Szedann** [comment]
@ClintFlames  do you mind if I use that to fix the recipes?

**Comment by KN13KOMETA** [comment]
@Szedann yep, use it.

**Comment by LosGinos** [comment]
@Szedann This might be a stupid question. But where, and how do I use your fix? I've been looking everywhere but I don't know where to place you files :(

**Comment by Szedann** [comment]
> @Szedann This might be a stupid question. But where, and how do I use your fix? I've been looking everywhere but I don't know where to place you files :( 

I did not make a fix, focusing on other things at the moment

**Comment by LosGinos** [comment]
Yeah, sorry I tagged the wrong person hehe. @ClintFlames I ask you instead This might be a stupid question. But where, and how do I use your fix? I've been looking everywhere but I don't know where to place you files :(

**Comment by ninjaeon** [comment]
> Yeah, sorry I tagged the wrong person hehe. @ClintFlames I ask you instead This might be a stupid question. But where, and how do I use your fix? I've been looking everywhere but I don't know where to place you files :(

Either look up a guide on how to load datapacks, or use a mod such as "[Open Loader](https://modrinth.com/mod/open-loader)" to do it (I prefer the latter)
